ci-operator/jobs/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master-presubmits.yaml

```diff
@@ -341,26 +341,25 @@ presubmits:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )build,?($|\s.*)
   - agent: kubernetes
-    always_run: false
+    always_run: true
     branches:
     - ^master$
     - ^master-
     cluster: build03
-    context: ci/prow/ci-index
+    context: ci/prow/ci-index-wmco-bundle
     decorate: true
     labels:
       ci.openshift.io/generator: prowgen
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
-    name: pull-ci-openshift-windows-machine-config-operator-master-ci-index
-    rerun_command: /test ci-index
-    skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
```

Comment:
Why are you removing this?

Reply:
My question was targeted at removing the `skip_if_only_changed:` line. In addition to the `always_run: true` implies this will run for even doc changes.